### DISCUSSION TITLE
Prevent modules talking to modules

### DISCUSSION THREAD

**Discussion by Mat-Moo** [nWidart/laravel-modules#1970] [Q&A]
I'm looking to play/implement modules to separate work in a new backend application that I'm about to start. I've used [Deptrac ](https://github.com/qossmic/deptrac)before to prevent the ability of one module talking directly to another (e..g module A talk to module B model) - Is this built into Laravel modules, or would I then need to configure Deptrac still as well?

**Comment by dcblogdev** [comment]
I think you would need to impelement Deptrac as there is nothing built in to prevent communication.
